core/ulog/README.md

```diff
@@ -1,8 +1,8 @@
 # Logging package
 
-ulog provides an API wrapper around the logging library (zap Logger)
-The logger is instantiated as logger with default options and can be configured
-via `Configure()` API and provided YAML configuration.
+`package ulog` provides an API wrapper around the logging library (zap Logger).
+`ulog` uses builder pattern to instantiate the logger. Using `LogBuilder` user can set up
```

Comment:
- builder pattern --> the builder pattern

Looks like some people capitalize builder pattern. I don't think we need to, but let's put "the" in front of it whenever it comes up.

- Is ``LogBuilder`` familiar to readers, or do you need to explain what that is too?
- Proposed rewrite of second sentence: "With ``LogBuilder``, you can set up configuration, inject the logger, and…"
- I'm not sure if "log level prior to logger initialization" means you're logging the level OR injecting the log level before initialization

Basically, you're saying that you can set up the logger and apply it to whatever you want before logging begins, right? If so, I'd say something like this:

"``LogBuilder`` is a configuration tool for ``ulog``. With ``LogBuilder``, you can configure the logger, inject it wherever you need it, and set the log level before logging begins."